.github/workflows/publish.yml

```diff
@@ -20,15 +20,16 @@ jobs:
         uses: actions/setup-python@v2
         with:
           python-version: ${{ matrix.python-version }}
-      
+
       #----------------------------------------------
-      #          install & configure poetry         
+      #          install & configure poetry
       #----------------------------------------------
       - name: Install Poetry
-        uses: snok/install-poetry@v1.1.1
-        with:
-          virtualenvs-create: true
-          virtualenvs-in-project: true
+        run: |
+          curl -sSL https://install.python-poetry.org \
+            | python3 - --version ${{ env.POETRY_VERSION }};
```

Comment:
we should also add the poetry version [env variable](https://github.com/Sage-Bionetworks/schematic/blob/develop/.github/workflows/test.yml#L29) to this workflow

Reply:
Oops, totally missed that.